source/indexes/compound-index.txt

```diff
@@ -0,0 +1,118 @@
+statements in your file:
+
+.. literalinclude:: /includes/indexes/compound-field.scala
+   :start-after: start-compound-index-imports
+   :end-before: end-compound-index-imports
+   :language: scala
+   :copyable:
+   :dedent:
+
+Use the ``createIndex()`` method to create a
+compound index. The following example creates an index in descending
+order on the ``runtime`` field and in ascending order on the ``year`` field:
+
+.. literalinclude:: /includes/indexes/compound-field.scala
+   :start-after: start-index-compound
+   :end-before: end-index-compound
+   :language: scala
+   :copyable:
+   :dedent:
+
+You can verify that the index was created by using the ``listIndexes()`` method.
+You should see an index for ``runtime`` and ``year`` in the list, as shown in the following output:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/indexes/compound-field.scala
+      :start-after: start-check-compound-index
+      :end-before: end-check-compound-index
+      :language: scala
+      :dedent:
+
+   .. output::
+      :visible: true
+
+      {"v": 2, "key": {"runtime": -1, "year": 1}, "name": "runtime_-1_year_1"}
+
+The following is an example of a query that is covered by the index
+created on the ``runtime`` and ``year`` fields:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/indexes/compound-field.scala
+      :start-after: start-index-compound-query
+      :end-before: end-index-compound-query
+      :language: scala
+      :dedent:
+
+   .. output::
+      :visible: false
+
+      {"_id":...,"runtime": 877,...,"title":"Taken",...,"year":2002,...}
+
+Additional Information
+----------------------
+
+To learn more about compound indexes, see :manual:`Compound
+Indexes </core/index-compound>` in the {+mdb-server+} manual.
+
+To view runnable examples that demonstrate how to manage indexes, see
+:ref:`scala-indexes`.
+
+API Documentation
+~~~~~~~~~~~~~~~~~
+
+To learn more about any of the methods discussed in this guide, see the following API
+documentation:
+
+- `Indexes.compoundIndex() <{+api+}/org/mongodb/scala/model/Indexes$.html#compoundIndex(indexes:org.mongodb.scala.bson.conversions.Bson*):org.mongodb.scala.bson.conversions.Bson>`__
+- `MongoCollection.createIndex() <{+api+}/org/mongodb/scala/MongoCollection.html#createIndex(key:org.mongodb.scala.bson.conversions.Bson,options:org.mongodb.scala.model.IndexOptions):org.mongodb.scala.SingleObservable[String]>`__
+- `MongoCollection.find() <{+api+}/org/mongodb/scala/MongoCollection.html#find[C](filter:org.mongodb.scala.bson.conversions.Bson)(implicite:org.mongodb.scala.bson.DefaultHelper.DefaultsTo[C,TResult],implicitct:scala.reflect.ClassTag[C]):org.mongodb.scala.FindObservable[C]>`__
+- `MongoCollection.listIndexes() <{+api+}/org/mongodb/scala/MongoCollection.html#listIndexes[C]()(implicite:org.mongodb.scala.bson.DefaultHelper.DefaultsTo[C,org.mongodb.scala.Document],implicitct:scala.reflect.ClassTag[C]):org.mongodb.scala.ListIndexesObservable[C]>`__
```

Comment:
Nit: 

```suggestion
- `compoundIndex() <{+api+}/org/mongodb/scala/model/Indexes$.html#compoundIndex(indexes:org.mongodb.scala.bson.conversions.Bson*):org.mongodb.scala.bson.conversions.Bson>`__
- `createIndex() <{+api+}/org/mongodb/scala/MongoCollection.html#createIndex(key:org.mongodb.scala.bson.conversions.Bson,options:org.mongodb.scala.model.IndexOptions):org.mongodb.scala.SingleObservable[String]>`__
- `find() <{+api+}/org/mongodb/scala/MongoCollection.html#find[C](filter:org.mongodb.scala.bson.conversions.Bson)(implicite:org.mongodb.scala.bson.DefaultHelper.DefaultsTo[C,TResult],implicitct:scala.reflect.ClassTag[C]):org.mongodb.scala.FindObservable[C]>`__
- `listIndexes() <{+api+}/org/mongodb/scala/MongoCollection.html#listIndexes[C]()(implicite:org.mongodb.scala.bson.DefaultHelper.DefaultsTo[C,org.mongodb.scala.Document],implicitct:scala.reflect.ClassTag[C]):org.mongodb.scala.ListIndexesObservable[C]>`__
```

Reply:
I'd say my personal preference would be only to include the object class name if there were multiple methods with the same name in different object classes (e.g. if there were two methods named `MongoCollection.functionName()` and `MongoClient.functionName()`.). I don't think there's an explicit standard to do it one way or the other, so I'll leave it to your judgment!

Reply:
Would prefer to keep as is, but seems to be the standard in other docs sets like [pymongo](https://www.mongodb.com/docs/languages/python/pymongo-driver/current/databases-collections/#api-documentation) and [kotlin sync](https://www.mongodb.com/docs/languages/kotlin/kotlin-sync-driver/current/aggregation/#api-documentation) to not include. I'll remove for consistency.